spec/integration/solr_instance_loader_spec.rb

```diff
@@ -43,6 +57,14 @@ class Foo < ActiveFedora::Base
       it "should not be mutable" do
         expect { subject.title = 'Foo' }.to raise_error ActiveFedora::ReadOnlyRecord
       end
+
+      it "loads the correct number of nested attributes", focus: true do
+        expect(subject.date_ids.count).to eq 2
+      end
+
+      it "loads the correct number of regular attributes", focus: true do
```

Comment:
Ugh, good catch! ha, AF's rspec wasn't even set up for this so I didn't notice it was there.